.github/workflows/edr-ci.yml

```diff
@@ -10,7 +10,6 @@ on:
   workflow_dispatch:
 
 env:
-  RUSTFLAGS: -Dwarnings
```

Comment:
This should not have any adverse side-effects, as we also manually pass this argument to `cargo clippy` which should fail if there are any warnings.

For reference:

`cargo clippy --workspace --all-targets --all-features -- -D warnings`